docs/pages/eas/workflows/syntax.mdx

```diff
@@ -268,32 +352,41 @@ maestro:
 Runs custom code and can use built-in EAS functions. Does not require a `type` field.
 
 ```yaml
-custom_job:
-  runs_on: linux-medium | linux-large | macos-medium | macos-large | linux-medium-nested-virtualization | linux-large-nested-virtualization # optional, default: linux-medium
-  environment: production | preview | development # optional, defaults to production
-  image: string # optional, defaults to 'default'. See https://docs.expo.dev/build-reference/infrastructure/ for a list of available images.
-  steps:
-    # ...
+jobs:
+  my_job:
+    steps:
+      # ...
 ```
 
-### `jobs.<job_id>.steps`
+### `jobs.<job_id>.runs_on`
 
-A job contains a sequence of tasks called `steps`. Steps can run commands or use an action.
+Specifies the worker that will execute the job.
 
 ```yaml
 jobs:
-  custom_job:
+  my_job:
     # @info #
-    steps:
-      - name: My first step
-        run: echo "Hello World"
+    runs_on: linux-medium | linux-large | linux-medium-nested-virtualization | linux-large-nested-virtualization | macos-medium | macos-large
     # @end #
 ```
 
+| Worker                             | vCPU | Memory (GiB RAM) | SSD (GiB) | Notes                                |
+| ---------------------------------- | ---- | ---------------- | --------- | ------------------------------------ |
+| linux-medium                       | 4    | 16               | 14        | Default worker.                      |
+| linux-large                        | 8    | 32               | 28        |                                      |
+| linux-medium-nested-virtualization | 4    | 16               | 14        | Allows running Android emulators.    |
+| linux-large-nested-virtualization  | 4    | 16               | 28        | Allows running Android emulators.    |
+| macos-medium                       | 5    | 12               | 85        | Runs iOS jobs, including simulators. |
+| macos-large                        | 10   | 20               | 85        | Runs iOS jobs, including simulators. |
+
+Note: For iOS builds and iOS simulator jobs, you must use a `macos-*` worker. For Android emulator jobs, you must use a `linux-*-nested-virtualization` worker.
+
```

Comment:
```suggestion
```

not sure if we need to mention this? If we do want to be proactive to users that may want to run Simulator / Emulator in custom jobs manually I think it may be good to mention that Maestro jobs pick the right `runs_on` automatically?